api_data.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import json
+import sys
+from urllib.request import urlopen
+
+def output_cve_id(type=None, name=None, version=None):
+    """Take as input, a package name, package version.
```

Comment:
Minor style point: start the doc on the next line as in
```
"""
Take as input, a package name....
```
Also it is best to start with what you return rather than what you accept